bigbluebutton-html5/imports/api/breakouts/server/handlers/breakoutClosed.js

```diff
@@ -0,0 +1,10 @@
+import clearBreakouts from '../modifiers/clearBreakouts';
+import { check } from 'meteor/check';
+
+export default function breakoutClosed({ payload }) {
+  const meetingId = payload.meetingId;
+
+  check(meetingId, String);
+
+  return clearBreakouts(payload.meetingId);
```

Comment:
You can use meetingId instead of payload.meetingId, especially since you assigned it to a variable